arrays/arrays.c

```diff
@@ -99,15 +99,25 @@ char *arr_read(Array *arr, int index) {
 void arr_insert(Array *arr, char *element, int index) {
 
   // Throw an error if the index is greater than the current count
-
+  if(index >= arr->count){
+    printf("Array capacity has been reached");
```

Comment:
Error message should say that the index is out of range. 

One possible change you might want to make to your arrays program is using stderr (standard error stream) and fprintf() (for writing data to a stream) to record and print error messages like so: ```fprintf(stderr, "IndexError: Index out of range")```.

 See: https://stackoverflow.com/questions/19870331/why-use-stderr-when-printf-works-fine. 

For example,